### PR TITLE
Revised Non Suppressed Register

### PR DESCRIPTION
**Pull request by slubwama**
This is a recreated because of too many commits on the Previous https://github.com/METS-Programme/openmrs-module-aijar/pull/327

Ticket https://trello.com/c/Bo3R6bba/661-add-changes-of-the-revised-viral-load-non-suppressed-register

### CODE REVIEW

**Comment by ssmusoke** [comment]
@slubwama Please can you attach the latest version of the Non-Supressed register that you used on the ticket

**Comment by slubwama** [comment]
@ssmusoke  Its done

File: api/src/main/resources/metadata/Custom_Concepts.xml [hunk 1]
```diff
@@ -4161,6 +4174,9 @@
     <concept_set concept_set_id="2182" concept_id="90315" concept_set="163169" sort_weight="2.0" creator="1" date_created="2016-07-28 13:26:18.0" uuid="2181AEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEE"/>
     <concept_set concept_set_id="2183" concept_id="163172" concept_set="163169" sort_weight="2.0" creator="1" date_created="2016-07-28 13:26:18.0" uuid="2182AEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEE"/>
     <concept_set concept_set_id="2184" concept_id="1305" concept_set="163157" sort_weight="2.0" creator="1" date_created="2016-07-28 13:26:18.0" uuid="2184AEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEE"/>
+    <concept_set concept_set_id="2404" concept_id="1305" concept_set="163151" sort_weight="2.0"  creator="52"  date_created="2017-11-14 13:07:48"  uuid="2404AEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEE"/>
```
Comment (ssmusoke) [inline api/src/main/resources/metadata/Custom_Concepts.xml]: @slubwama This is the creator who was causing issues for me - double check and ensure that all creators have a value of 1

**Comment by ssmusoke** [comment]
@slubwama 

1. Regimen at VL Test, Regimen after IAC Session - need to include DTG

2. What are the required fields for this form which must be entered? Recommendations:
- Date Results Received at Health Facility
- VL Results at Initiation
- Date Results Given to Client
- Regimen at VL Test
- Start Date

3. Validation Rules:
- Date Results Received 
- IAC session dates - should be after date of Registration
- Start Date should be on or after date of Registration
- Date Results given to client must be after Date Results Received at the health facility
- Date of Registration must be after Date Results Received at Health Facility
- Date of VL Repeat must be after the latest date of IAC session
- Date of Results Received (VL after IAC) must be after date of VL Repeat
- Decision Date must be after Date Results Received (VL after IAC session)

4. Date fields allow the selection of dates in the future 

5. IAC sessions - the form includes 6 sessions, I suggest we put the ability to add up to 10 sessions to cater for those who are under IAC for much longer

**Comment by slubwama** [comment]
@ssmusoke I think "6" should be fixed with the widget other than in this PR Requesting that you remove it from the comment.

**Comment by ssmusoke** [comment]
@slubwama the issue disappears when the encounter is deleted so need to find the root cause which is the encounter data

**Comment by slubwama** [comment]
@ssmusoke 
"Date fields allow the selection of dates in the future" This happens in all our other forms but don't allow saving of the date. Thats how we have designed it unless you want to start a new standard.

**Pull request merged**